[*] Running ...
[*] Dataset loaded: C:\Development\cs5491-artificial-intelligence/problems/tsp_constructive\dataset\train50_dataset.npy with 64 instances.
Traceback (most recent call last):
  File "C:\Development\cs5491-artificial-intelligence/problems/tsp_constructive/eval.py", line 82, in <module>
    obj = eval_heuristic(node_positions[i])
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Development\cs5491-artificial-intelligence/problems/tsp_constructive/eval.py", line 42, in eval_heuristic
    next_node = select_next_node(
                ^^^^^^^^^^^^^^^^^
  File "C:\Development\cs5491-artificial-intelligence\problems\tsp_constructive\gpt.py", line 20, in select_next_node_v2
    next_unvisited_distances = np.append(distance_matrix[next_node][unvisited_node_list], [distance_matrix[next_node][current_node], distance_matrix[next_node][destination_node]])
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: operands could not be broadcast together with shapes (48,) (2,) 
